### DISCUSSION TITLE
can we change bootstrap primary colors during runtime?

### DISCUSSION THREAD

**Discussion by elijah-lucian** [twbs/bootstrap#32806] [Q&A]
pretty simple question. If I want to make a design system, and I want to change the colors based on an API call in an app, how do i change the primary colors? 

The sass is compiled at this point, and changing the --primary variable does nothing. 

Is there an approach I am not thinking about?

**Comment by mdo** [comment]
We expose CSS variables for additional customization, but at this time, we do not support full runtime customization for them.
